fileresult.txt

```diff
@@ -0,0 +1 @@
+writer: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=2f895e6693210c91ca23a1f49a239119b021fa94, for GNU/Linux 3.2.0, not stripped
```

Comment:
@AbhishekKoppaCU looks like the file didn't get cross compiled here, still `x86-64` type.